src/lib/utils/embed.ts

```diff
@@ -0,0 +1,83 @@
+import type { APIEmbed, BaseMessageOptions, InteractionReplyOptions } from 'discord.js';
+import {
+	ARROW_RIGHT,
+	BOT_AVATAR,
+	BOT_COLOR,
+	BOT_NAME,
+	FAIL,
+	IS_CUSTOM_BOT,
+	SUCCESS,
+} from '../../helpers/provide/environment';
+
+type UniversalMessageOptions = Omit<BaseMessageOptions, 'flags'>;
+type UniversalInteractionOptions = Omit<InteractionReplyOptions, 'flags'>;
+
+export function generateEmbedWithDefault(embed: APIEmbed): APIEmbed {

```

Comment:
rename to `generateDefaultEmbed`
```suggestion
export function generateDefaultEmbed(embed: APIEmbed): APIEmbed {
```